tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -93,6 +98,40 @@ describe('util/extension-runners/chromium', async () => {
 
     await new Promise((resolve) => wsClient.on('open', resolve));
 
+    // Clear console stream from previous messages and start recording
+    consoleStream.stopCapturing();
+    consoleStream.flushCapturedLogs();
+    consoleStream.startCapturing();
+    // Make verbose to capture debug logs.
+    consoleStream.makeVerbose();
+
+    // Emit a fake socket object as a new wss connection.
+    const socket = await new Promise((resolve) => {
```

Comment:
The "error" event is synchronously added in the "connection" event. Since there is no asynchronicity, let's not use `await` here. After all, if the "error" event listener was somehow added asynchronously, then there is a chance that the "error" event is triggered before the listener existed, and cause an uncaught error.

```javascript
const fakeSocket = new EventEmitter();
const socketOnSpy = sinon.spy(fakeSocket, 'on');
runnerInstance.wss.emit('connection', fakeSocket);
sinon.assert.calledOnce(socketOnSpy);
// TODO:maybe check that the spy was called with "error"?
socket.emit('error', ...);

```